### PR TITLE
5772 case in detailed view table

### LINKED ISSUE #5772: Create an automation test where the data from a case is checked in the Detailed view table
```gherkin
  Scenario: Check all fields from the created Case in the Detailed view table
    Given I create a case using the api
    Given I login with the user
    When I am accessing the Case directory
    And I switch to the Detailed view
    Then I search for the created case in the Detailed View
    And I am checking if all the fields are correctly displayed
```
## Additional info
- Check if #5665 is reusable here

### PR DESCRIPTION
**Pull request by alinapopaiustina**
Fixes #5772

image
Please review!
Thank you!

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    // because API request is sending local GMT and UI displays GMT+2 (server GMT)
+    outputFormat.setTimeZone(TimeZone.getTimeZone("GMT+2"));
+
+    // inputFormat is the format of teh dateTime as read from API created case
+    DateFormat inputFormat = new SimpleDateFormat("EEE MMM dd HH:mm:ss zzz yyyy", Locale.US);
+    Date parsedDate = null;
+    try {
+      parsedDate = inputFormat.parse(dateTimeString);
+    } catch (ParseException e) {
+      e.printStackTrace();
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:195]: add log to print the exception

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+  public String getFollowUpUntilCaseDate(String dateOfReportDateDateTime) {
+    SimpleDateFormat outputFormat = new SimpleDateFormat("M/dd/yyyy");
+    DateFormat inputFormat = new SimpleDateFormat("MM/dd/yyyy h:mm a");
+    ZoneId defaultZoneId = ZoneId.systemDefault();
+    Date parsedDate = null;
+    try {
+      parsedDate = inputFormat.parse(dateOfReportDateDateTime);
+    } catch (ParseException e) {
+      e.printStackTrace();
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:208]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    }
+    return outputFormat.format(parsedDate);
+  }
+
+  public String getFollowUpUntilCaseDate(String dateOfReportDateDateTime) {
+    SimpleDateFormat outputFormat = new SimpleDateFormat("M/dd/yyyy");
+    DateFormat inputFormat = new SimpleDateFormat("MM/dd/yyyy h:mm a");
+    ZoneId defaultZoneId = ZoneId.systemDefault();
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:203]: move variable in class

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/CaseDirectoryPage.java [hunk 4]
```diff
@@ -22,6 +22,17 @@
...
+  public static final By CASE_DIRECTORY_DETAILED_RADIOBUTTON =
+      By.cssSelector("div#casesViewSwitcher span:nth-child(2) > label");
+  public static final By CASE_DIRECTORY_DETAILED_PAGE_FILTER_INPUT =
+      By.cssSelector("input#nameUuidEpidNumberLike");
+  public static final By CASE_DIRECTORY_DETAILED_PAGE_APPLY_FILTER_BUTTON =
+      By.cssSelector("div#actionApplyFilters");
+  public static final By CASE_DETAILED_TABLE_DATA = By.tagName("td");
+  public static final By CASE_DETAILED_COLUMN_HEADERS =
+      By.cssSelector("thead .v-grid-column-default-header-content");
+  public static final By CASE_DETAILED_FIRST_TABLE_ROW =
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/CaseDirectoryPage.java:34]: should be ROWS if we're using get elements and get more than one, even if for test purpose we're taking the first one

Comment (alinapopaiustina) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/CaseDirectoryPage.java:34]: :) in a previous PR I had a comment to call it first row... will do it. :)

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java (hunk 3 above)
Comment (alinapopaiustina) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:203]: This variable is already in class: apiState.getCreatedCase().getReportDate()

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java (hunk 2 above)
Comment (alinapopaiustina) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:208]: addressed

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java (hunk 1 above)
Comment (alinapopaiustina) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java:195]: addressed

**Comment by alinapopaiustina** [comment]
@rdutu-vg please see addressed review remarks! Thank you! :)

**Review by rdutu-vg — APPROVED** [review]

**Review by VGitVlad — APPROVED** [review]

**Review by Mihai-Bota — APPROVED** [review]

**Pull request merged**